roboflow/core/project.py

```diff
@@ -145,14 +145,13 @@ def __image_upload(self, image_path, hosted_image=False, split="train"):
         # If image is not a hosted image
         if not hosted_image:
 
-            project_name = self.id.rsplit("/")[1]
             image_name = os.path.basename(image_path)
 
             # Construct URL for local image upload
             self.image_upload_url = "".join(
                 [
-                    "https://api.roboflow.com/dataset/",
```

Comment:
In some cases we may want to change the API_URL from ENV, this seemed to have stuck around as hard coded